### ISSUE TITLE
Implement BiLSTM baseline

### ISSUE THREAD

**Issue by rchan26** [ttseriotou/sig-networks#5]
- [x] Tidy up Talia's code for BiLSTM for classification (to apply on the history)
- [x] Implement function to hyperparameter tuning for BiLSTM. Allow tuning for:
  - [x] window-size
  - [x] hidden state size
  - [x] dropout
  - [x] learning rates
